packages/test_packages_common.py

```diff
@@ -24,7 +24,7 @@ def registered_packages_meta():
     """Returns a dictionary with the contents of `meta.yaml`
     for each registered package
     """
-    packages = registered_packages
+    packages = registered_packages()
```

Comment:
Pretty sure this is a fix. You can't iterate a raw function...

Reply:
Maybe this is dead code then?

Reply:
Probably?

Reply:
Yes, it's never called you can remove it.